spec/models/diary_entry/steps/log_entry_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+RSpec.describe Diary::Steps::LogEntry do
+  include_context "wizard step"
+  it_behaves_like "a wizard step"
+
+  context "attributes" do
+    it { is_expected.to respond_to :entry }
+  end
+
+  describe "entry" do
+    it { should validate_length_of(:entry).is_at_least(10) }
```

Comment:
`is_expected.to` instead of `should` 🙏  for consistency. The same is in the other step spec.

Reply:
Done